.github/CODEOWNERS.txt

```diff
@@ -0,0 +1,4 @@
+# For details on CODEOWNERS file, see
```

Comment:
Should CODEOWNERS have a ".txt" extension?

Reply:
I believe it should be `md`, for markdown, but maybe GH supports `txt` as well. :man_shrugging:

Reply:
oops, I missed that. Thanks for catching it!